app/models/message.rb

```diff
@@ -27,6 +27,7 @@
 #  index_messages_on_additional_attributes_campaign_id  (((additional_attributes -> 'campaign_id'::text))) USING gin
 #  index_messages_on_content                            (content) USING gin
 #  index_messages_on_conversation_id                    (conversation_id)
+#  index_messages_on_created_at                         (created_at)
```

Comment:
Should we add created_at + account_id rather than just the created_at?

Reply:
Let me try that

Reply:
An index on `conversation_id`, `account_id`, `message_type` and `created_at` does speed up this specific query. We can keep both. I'll merge this for now and create another PR with this new indexes

## results
Before: https://explain.dalibo.com/plan/da7c4h9e9161ba64
After: https://explain.dalibo.com/plan/b2c9ece7247120c8